source/includes/fact-verifier-limitations.rst

```diff
@@ -0,0 +1,33 @@
+
+The embedded verifier has the following limitations:
+
+- ``mongosync`` stores the verifier state in memory, which can result
+  in a significant memory overhead. To run verification checks, ``mongosync``
+  requires 1 GB of memory for every 20 million documents.
+
+- Verification is not resumable. If a user stops and then starts ``mongosync``
+  for any reason (whether they do so by pausing and resuming the sync or by stopping 
```

Comment:
Consider rewrite to simplify long statement.

Reply:
Fixed